config/vufind/sitemap.ini

```diff
@@ -7,10 +7,24 @@
 ;   individual sitemap files
 [Sitemap]
 
-;  Frequency tells the site indexer (aka Google, etc) how frequently you
+;  Frequency tells the site indexer how frequently you
 ;    expect the content of the given URL entry to be updated.  The options are:
-;    always, hourly, daily, weekly, monthly, yearly and never
-frequency      = weekly
+;    always, hourly, daily, weekly, monthly, yearly and never.
+;
+;    Note that the effect of this setting is highly depending on the indexer,
+;    e.g. Google will ignore this setting completely and prefer the "lastmod"
+;    information instead, which will automatically be exported if
+;    Change Tracking is available.
+;
+;    It is also recommended to rather increase this setting if you have a high
```

Comment:
Maybe, rather than putting these details here, it would be better to link to the https://vufind.org/wiki/administration:search_engine_optimization wiki page and make sure it's up to date and accurate.

Reply:
OK, so i added a reference to the Wiki page and moved most of the notes to the section Best Practices => Understanding Crawling Budgets in the Wiki (maybe the exact VuFind Version should be mentioned there when lastmod will be available after this PR has been merged.)